v0/am/transform.js

```diff
@@ -62,9 +62,15 @@ function fixSessionId(payload) {
     ? stringToHash(payload.session_id)
     : -1;
 }
- function addMinIdlength() {
-   return {"min_id_length":1}
- }
+
+function fixVersion(payload, message) {
```

Comment:
can we change only the config jsons and not add here ?
like:
"context.app.build": "app_version", ==> "context.app. version": "app_version"

Reply:
The default should be `context.app.build` but for android specifically, it should be `context.app.version`

Reply:
👍